pydantic/_migration.py

```diff
@@ -22,6 +22,8 @@
     'pydantic.decorator:validate_arguments': 'pydantic.deprecated.decorator:validate_arguments',
     'pydantic.class_validators:validator': 'pydantic.deprecated.class_validators:validator',
     'pydantic.class_validators:root_validator': 'pydantic.deprecated.class_validators:root_validator',
+    'pydantic.utils:lenient_isinstance': 'pydantic._internal._utils:lenient_isinstance',
+    'pydantic.utils:lenient_issubclass': 'pydantic._internal._utils:lenient_issubclass',
```

Comment:
Can we change this to import from `pydantic.v1.utils`, then remove them from `pydantic._internal._utils`?